services/ir/artifacts.mk

```diff
@@ -31,14 +31,22 @@ get.cli:
 	@mkdir -p ./vendor
 
 ifeq (${NEOFS_CLI_PATH},)
-	@echo "⇒ Download NeoFS CLI binary from ${NEOFS_CLI_URL}"
-	@curl \
-		-sSL "${NEOFS_CLI_URL}" \
-		-o ${NEOFS_CLI_ARCHIVE_FILE} 
-	@tar -xvf ${NEOFS_CLI_ARCHIVE_FILE} -C ./vendor | xargs -I {} \
-		mv ./vendor/{} ${NEOFS_CLI_FILE}
-	@rm ${NEOFS_CLI_ARCHIVE_FILE}
+    ifneq (${NEOFS_CLI_URL},)
+		@echo "⇒ Download NeoFS CLI binary from ${NEOFS_CLI_URL}"
+		@curl \
+		    -sSL "${NEOFS_CLI_URL}" \
+		    -o ${NEOFS_CLI_ARCHIVE_FILE}
+		@tar -xvf ${NEOFS_CLI_ARCHIVE_FILE} -C ./vendor | xargs -I {} \
+			mv ./vendor/{} ${NEOFS_CLI_FILE}
+		@rm ${NEOFS_CLI_ARCHIVE_FILE}
+    else
+	    @echo "⇒ NEOFS_CLI_URL is empty."
```

Comment:
Both NEOFS_CLI_URL and NEOFS_CLI_PATH are empty here.

Reply:
> Both NEOFS_CLI_URL and NEOFS_CLI_PATH are empty here.

Yes, this is expected behavior.
For example, if the user builds part of dev-env locally, this can happen.
This is not an error, so only the message about empty variables is added.

Reply:
My point is that your message should be clear about that.